packages/angular_devkit/core/node/resolve.ts

```diff
@@ -90,7 +90,7 @@ export interface ResolveOptions {
 
   /**
    * Whether or not to preserve symbolic links. If false, the actual paths pointed by
-   * the symbolic links will be used. This defaults to true.
+   * the symbolic links will be used. Defaults to `true` if NodeJS option --preserve-symlinks is set.
    */
   preserveSymlinks?: boolean;
```

Comment:
This is not effected by the change you did, also this is deprecated logic which is no longer used internally.

Reply:
Right, added to the scriptions of where I could find "Do not use the real" instead